.github/actions/build-electron/action.yml

```diff
@@ -36,7 +36,7 @@ runs:
       shell: bash
       if: ${{ inputs.target-arch == 'x64' && inputs.target-platform == 'macos' }}
       run: |
-        GN_APPENDED_ARGS="$GN_EXTRA_ARGS v8_snapshot_toolchain=\"//build/toolchain/mac:clang_x64\""
```

Comment:
Not blocking the PR on this, but I think we originally needed "//build/toolchain/mac:clang_x64\" for mksnapshot to be the correct arch 👀 We may just want to check that we don't have a similar problem with mksnapshot

Reply:
@VerteDinde GN_EXTRA_ARGS isn't used for mksnapshot

Reply:
Hmm maybe I'm wrong here

Reply:
@VerteDinde I don't see GN_EXTRA_ARGS being used for mksnapshot, but with my change if you look at https://github.com/electron/electron/actions/runs/10163398663/artifacts/1755804915, the mksnapshot binary there is arm64.

Reply:
@VerteDinde figured it out. Build tools reads GN EXTRA ARGS: https://github.com/electron/build-tools/blob/main/src/e-build.js#L23